amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -1,72 +1,102 @@
+. Drag the operation onto the Studio canvas to the right of the source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target Amazon S3 system. Amazon S3 supports basic and role authentication.
+
+To configure the global element for Amazon S3 Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, in *Connection*, select the authentication method to configure:
+  * <<basic-authentication,Basic Authentication>>
+  * <<role-authentication,Role Authentication>>
 
-. To create a global element for the connector, click the *Global Elements* tab at the base of the canvas, and then click *Create*.
-. In the *Choose Global Type* window, expand *Connector Configuration*, and click *Amazon S3 Configuration*.
-. Click *OK*.
-. Select *Amazon S3 Configuration (Configuration)*, and click *Edit*.
-. Specify the global element properties:
 +
-image::amazon-s3-01.png[Global Element Properties]
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
 +
-[%header%autowidth.spread]
-|===
-|Parameter |Description
-|Name |Name used to reference the connector instance
-|Access Key |Alphanumeric text string that uniquely identifies the user who owns the AWS account
-|Secret Key |Key that plays the role of a password
-|Try Default AWS Credentials Provider Chain |Indicates whether to use temporary credentials
-|Region Endpoint |URL entry point for a web service
-|Session Token |The session token used to validate the temporary security credentials
-|===
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
 +
 You can either enter your credentials into the global configuration properties or reference a configuration placed in a file in the `src` folder of your project file that contains these values. For simpler maintenance and better reuse of your project, use a configuration file.
 +
-. If you are:
-+
 . If you are using the default Amazon S3 storage, leave the default entries for *Proxy*, *Security*, and *Advanced* tabs.
```

Comment:
This information should be integrated or attached to the previous steps. Maybe you could add a paragraph after the steps like this:

 Note the following when you configure a global element for Amazon S3:

* info from this paragraph
* info from the next paragraph

Reply:
Good idea.